### PR TITLE
Update nesting.rst

### PR DESCRIPTION
**Pull request by JitQL**
These changes are, more or less, a conclusion of what's been learned before, that I think help understand the info on the page. I'm not absolutely sure they are right, but I do think we need them as an introduction to the comprehensive info that comes next. It helps get the overall picture without exhaustive info.

It's really fundamental knowledge and one tends to get lost on these concepts here, as well as on lesson 02.

I think the need for having 3 brackets as input for viz nodes in lesson 02 needs better explanation and that explanation can be either there on Lesson 02 or here in this page as the final example is a specific example of a single face, and I found it hard to understand at first without comparing it with a regular example side by side (I might still haven't got it right though). I will deal with that later.

## Addressed problem description

Please describe what problem does this PR solve. If it was already was discussed in an issue, it is enough to give a link as #number.
If this adds a new feature (a new node, for example), please state why (for what cases) do you think it is needed.

## Solution description

Please describe how do you intend to solve the problem: change some method, or introduce new class, or whatever.

## Preflight checklist

Put an x letter in each brackets when you're done this item:

- [ ] Code changes complete.
- [ ] Code documentation complete.
- [ ] Documentation for users complete (or not required, if user never sees these changes).
- [ ] Manual testing done. 
- [ ] Unit-tests implemented.
- [ ] Ready for merge.

### CODE REVIEW

File: docs/nesting.rst [hunk 1]
```diff
@@ -3,18 +3,49 @@ Nesting
...
+
+``edges_list = [(0, 1), (1, 2), (2, 3)]``
+
+So basically, sverchok works by using nodes to creat lists of values, then using other nodes to evaluate those lists and generate new lists. Eventually the results of the final list will be used to generate the final output that will be passed to the viewport with a viewer node.
+
+This is why the nesting concept is fundamental in order to use Sverchok.
+
+Let's look at some examples, (not necessarily outputting just one thing): 
+
+NOTE to zeffii, what do you mean with this: (not necessarily outputting just one thing)?
```
Comment (zeffii) [inline docs/nesting.rst:36]: num things | descriptor
----|----
"one thing" | would be a socket that outputs a single collection, so `[[item1, item2, item3]]`
"two things" | would be a socket that outputs two collections `[[item1, item2, item3], [item1, item2, item3]]`

**Comment by zeffii** [comment]
these are fine, both PR as OK.  

My only concern is the use of the words "so" (at the start of any sentence) and "basically" (anywhere), drop those and we're golden.  

https://wynnnetherland.com/linked/2014010902/words-to-avoid-in-tech-writing

**Comment by JitQL** [comment]
That is a very useful comment and advice. I think it applies to much more than tech and learning. I will remove the words and try to think on that advice in other contexts too. Thanks!

I understood the explanation sbout one or two things too.

**Comment by zeffii** [comment]
I like to stick information in a table when it has a chance to remove the complexity of information being covered. But hey, the more examples people read the more they will pick up on patterns, even if we fail in our explanation of individual parts/examples. When people recognize the patterns, that's when they've learned what we're trying to get across.

keep going!

File: docs/nesting.rst (hunk 1 above)
Comment (zeffii) [reply docs/nesting.rst:38]: fair comment btw, if that line didn't make sense on the first reading. The comment in parens could be dropped, i don't think it was helpful the way i phrased it originally.

**Pull request merged**